Spokestack/TFLiteWakewordRecognizer.swift

```diff
@@ -150,7 +150,7 @@ import TensorFlowLite
         
         // Tracing
         self.traceLevel = c.tracing
-        if self.traceLevel.rawValue >= Trace.Level.DEBUG.rawValue {
+        if self.traceLevel.rawValue <= Trace.Level.DEBUG.rawValue {
```

Comment:
There are a lot of fields here just for tracing that could probably be collapsed into a single data structure. Also, the trace level checks throughout the file oscillate between `< PERF` and `<= DEBUG`; it seems like you just want `== DEBUG` since it's the lowest level.

I think a bigger question might be that since the code for this is settled, is this file-based debugging of every piece of the wakeword code still helpful? Certainly some tracing for resets/activations is worth leaving in, but are the finer details useful for normal debugging?

Reply:
Good catch with the `< PERF` and `<= DEBUG` oscillations, that was crufty code.

This code is only settled for the current implementation. The implementation has already changed twice, with a third (multi-wakeword) in flight, so it is useful to keep the most intensive level of tracing messages for debugging purposes intact.

Reply:
`<= .DEBUG` preserves future compatibility if the tracing levels change.

Reply:
Yeah, I considered the possibility of changing levels, but that seemed highly unlikely. Fair point about the upcoming wakeword addition.

Reply:
I have thought that an `INTERNAL` level below `DEBUG` might be suitable for some of tracing messages in this class specifically.